spark-indexer/src/test/java/org/uniprot/store/spark/indexer/proteome/converter/DatasetProteomeEntryConverterTest.java

```diff
@@ -0,0 +1,813 @@
+package org.uniprot.store.spark.indexer.proteome.converter;
```

Comment:
Add a negative test of unknown citationType to cover the line where we are throwing RuntimeException